tools/targets/PSOC6.py

```diff
@@ -129,33 +137,61 @@ def complete_func(message_func, elf0, hexf0, hexf1=None, dest=None):
 
 
 # Find Cortex M0 image.
-def find_cm0_image(toolchain, resources, elf, hexf, hex_filename):
-    if hex_filename is None:
+def find_cm0_image(message_func, resources, m0hex_filename):
```

Comment:
Who's calling  t his function? I can see in the old tools __init__.py , that one we do not need neither use so this function is not invoked?

Reply:
Remnant from the previous refactoring. Cleaning now.
Thanks